smartcar-auth/src/main/java/com/smartcar/sdk/VehicleInfo.java

```diff
@@ -0,0 +1,130 @@
+package com.smartcar.sdk;
+
+/**
+ * A class that creates a custom VehicleInfo object, which can be used
+ * when generating an authentication URL.
+ */
+public class VehicleInfo {
+    private String vin;
+    private String make;
+    private String model;
+    private Integer year;
+
+    /**
+     * Assigns optional and required properties on the VehicleInfo object.
```

Comment:
What does this mean?

Reply:
Not sure, was part of the old class, I have since then refactored it to be a non builder class, new comments.

Reply:
Refactored back to being builder.